README.md

```diff
@@ -4,6 +4,23 @@ A genome "sketch" is a set of preprocessed data that makes it fast to compare ge
 
 This is a KBase dynamic (persistent) service that takes workspace IDs pointing to Assemblies, Genomes, or Reads and returns a list of similar genomes.
 
+## Usage
+
+### Basic requests
+
+From the terminal, you can use the `curl` command to access the sketch service. Since the sketch serivce is a KBase dynamic service, you will have to find its active URL by searching for it here: "https://ci.kbase.us/#catalog/services".
+
+The service accepts POST requests with a JSON payload with the following parameters:
+
+```
+{
+  "params": {
+    "ws_ref": <workspace reference>
```

Comment:
another thing I forgot: We didn't update the tests to reflect the new param structure. Will do on master